fxgl-entity/src/main/kotlin/com/almasb/fxgl/entity/level/tiled/TilesetLoader.kt

```diff
@@ -134,6 +145,20 @@ class TilesetLoader(private val map: TiledMap, private val mapURL: URL) {
 
             log.debug("Writing to buffer: dst=${x * map.tilewidth},${y * map.tileheight}, w=$w,h=$h, src=$srcx,$srcy")
 
+            if (isHorizontallyFlipped) {
+                sourceImage = flipHorizontally(Texture(sourceImage).subTexture(Rectangle2D(srcx.toDouble(), srcy.toDouble(), w.toDouble(), h.toDouble())).image)
```

Comment:
Could you explain why this is needed? Would `sourceImage = flipHorizontally(sourceImage)` not work?

Reply:
Yes this is why, but only when the `Tileset.isSpriteSheet` is true. I can fix this to simply flip when loading from a collection of images and grab this subTexture and then flip when loaded from sprite sheet.